client/src/components/friendModal/FriendItem.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { ListItem, ListItemAvatar, ListItemText, Button, Avatar, makeStyles } from '@material-ui/core';
+import { blue } from '@material-ui/core/colors';
+
+const useStyles = makeStyles({
+  addFriendButton: {
+    borderRadius: 25,
+    color: 'white',
+    backgroundColor: '#62cd4d',
+  },
+  removeFriendButton: {
+    borderRadius: 25,
+    color: 'white',
+    backgroundColor: '#D5D5D5',
```

Comment:
dont use raw color value here. whether its hex or string try to use it inside a theme.js .. see the way I use it...

Reply:
Ok, I'll change it to hex and put it in theme.js